package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Brackets",
-    "version": "1.8.0-0",
-    "apiVersion": "1.8.0",
+    "version": "1.9.0-0",
+    "apiVersion": "1.9.0",
```

Comment:
Needs rebase against master so we don't have these duplicate files, `git fetch upstream master && git rebase upstream/master`.